elements/pfe-button/BaseButton.ts

```diff
@@ -127,28 +20,21 @@ export type ButtonVariant = (
 export abstract class BaseButton extends LitElement {
   static readonly styles = [styles];
 
+  static readonly formAssociated = true;
+
+  static readonly shadowRootOptions: ShadowRootInit = { ...LitElement.shadowRootOptions, delegatesFocus: true };
+
   /** Disables the button */
-  @observed
   @property({ reflect: true, type: Boolean }) disabled = false;
 
-  /**
-   * Changes the style of the button.
-   * - Primary: Used for the most important call to action on a page. Try to
-   *   limit primary buttons to one per page.
-   * - Secondary: Use secondary buttons for general actions on a page, that
-   *   don’t require as much emphasis as primary button actions. For example,
-   *   you can use secondary buttons where there are multiple actions, like in
-   *   toolbars or data lists.
-   * - Tertiary: Tertiary buttons are flexible and can be used as needed.
-   */
-  @property({ reflect: true }) variant: ButtonVariant = 'primary';
```

Comment:
Should this be an abstract field?  Does `rh-button` also need a variant property?